src/components/venue-socials/venue-socials.ce.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div v-if="venue == 'courtyard'">
+    <ActivitiesContacts
+      name="The Courtyard"
+      instagram="thecourtyard_yorksu"
+      facebook="courtyardyusubars"
```

Comment:
Do we even use FB for these venues? can double check with comms/marketing can we also check if they want the events email on this too?